lib/PuppeteerSharp/PuppeteerSharp.csproj

```diff
@@ -29,13 +29,12 @@
   </PropertyGroup>
   <Import Project="../Common/CommonProps.props" />
   <ItemGroup>
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="8.0.0" />
```

Comment:
The two new libaries added here should most likely only be included for `netstandard2.0`